firebase.json

```diff
@@ -506,8 +506,13 @@
         "type": 301
       },
       {
-        "source": "/marketplace",
-        "destination": "https://cortex.marketplace.pan.dev/marketplace",
```

Comment:
We still need this is for this case https://xsoar.pan.dev/marketplace/

Reply:
The client-side redirect (src/pages/marketplace/index.js) will take care of `/marketplace` and `/marketplace/?<url query args>`.